src/theme.ts

```diff
@@ -0,0 +1,18 @@
+export const PrimaryTextColor = '#21232c'
+export const SecondaryTextColor = '#8b97ab'
+
+export const BalanceHeroCryptoColor = '#ff9b73'
+export const BalanceHeroUsdValueColor = PrimaryTextColor
+export const BalanceHeroUsdIconColor = '#4d4f56'
+export const BalanceHeroCryptoValueColor = SecondaryTextColor
+
+export const ListBackgroundColor = '#f9f9fb'
+
+export const OpenColor = '#53d0ab'
+export const ClosedColor = '#fca944'
+export const SettledColor = '#676ad7'
+
+export const ListItemTextColor = '#182e57'
+export const ListItemDividerColor = '#e9eaed'
+
+export const BlueLinkColor = '#00abe9'
```

Comment:
We probably should not put name of the color in the name, as changing the color will require to change the name as well.